media/css/privacy/privacy-firefox-simple.scss

```diff
@@ -0,0 +1,186 @@
+        max-width: 640px;
+        margin: 0 auto;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Policy header
+
+.privacy-title {
+    margin-bottom: $spacing-lg;
+
+    h1 {
+        @include text-title-xs;
+        font-weight: bold;
+    }
+
+    a {
+        color: inherit;
+        text-decoration: none;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Policy header intro
+
+.privacy-lead-in {
+    border-bottom: $border;
+    clear: both;
+    margin-bottom: $layout-sm;
+    padding-bottom: $layout-xs;
+
+    h2 {
+        @include text-title-xs;
+        margin-top: $spacing-2xl;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Main policy content
+
+.privacy-body {
+    > div > section > section {
+        border-bottom: $border;
+    }
+
+    h2 {
+        @include text-title-xs;
+        margin-bottom: $layout-sm;
+    }
+
+    h3 {
+        @include text-body-lg;
+        margin: $layout-sm 0;
+    }
+
+    h4 {
+        @include text-body-md;
+        margin: $layout-sm 0;
+    }
+
+    hr {
+        display: none;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Privacy choices widget
+
+.data-choices {
+    @include clearfix;
+    background-color: $color-marketing-gray-20;
+    border-radius: $border-radius-md;
+    margin-top: $spacing-xl;
+    padding: $spacing-sm;
+    position: relative;
+    text-align: center;
+
+    p {
+        @include text-body-md;
+        display: inline-block;
+        margin: 0;
+        padding: $spacing-sm 0;
+    }
+
+    .mzp-c-button {
+        display: block;
+        margin: $spacing-md auto 0;
+    }
+
+    @media #{$mq-md} {
+        @include grid-column-gap($spacing-lg);
+        display: grid;
+        grid-template-columns: 4fr 1fr;
+
+        @include bidi(((text-align, left, right),));
+
+        .mzp-c-button {
+            margin: 0;
+        }
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Policy footer
+
+.privacy-footnote {
+    @include text-body-md;
+    border-top: $border;
+    margin-top: $spacing-2xl;
+    padding: $spacing-2xl 0;
+
+    h2 {
+        @include text-title-xs;
+    }
+
+    h3 {
+        @include visually-hidden;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Summary and details widget
+
+.format-headings .privacy-body {
+    @include details;
+
+    .is-summary {
+        button {
+            @include summary;
+            @include bidi(((padding-left, 0, padding-right, 0),));
+        }
+
+        button[aria-expanded='true']::before {
+            @include summary-open;
+        }
+
+        + div {
+            margin-top: $spacing-sm * -1;
+            margin-bottom: $spacing-xl;
+        }
+    }
+}
+
+.format-paragraphs .privacy-body {
```

Comment:
As far as I can tell none of the terms or notices use the `base-notice-paragraphs` template and `.format-paragraphs` styles can be safely omitted. (That template collapses content into [a series of "learn more" buttons](https://www.allizom.org/en-US/privacy/websites/)).